### PR TITLE
Rn cl cumulocity 10.18.570.0

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

**Comment by BeateRixen** [comment]
Please review and approve the entries for your team: 
@Oezge-Akin > 4 tickets
@sagscmi > 2 tickets
@martin-sag > 1 ticket
@l2p- > 1 ticket

File: content/change-logs/device-management/cumulocity-10-18-540-13-fixed-an-issue-in-the-realtime-api.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Fixed an issue in the realtime API "
+product_area: Device management & connectivity
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
```
Comment (sagscmi) [inline content/change-logs/device-management/cumulocity-10-18-540-13-fixed-an-issue-in-the-realtime-api.md:10]:
```suggestion
    label: REST API
```

File: content/change-logs/device-management/cumulocity-10-18-540-13-fixed-an-issue-in-the-realtime-api.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Fixed an issue in the realtime API "
+product_area: Device management & connectivity
```
Comment (sagscmi) [inline content/change-logs/device-management/cumulocity-10-18-540-13-fixed-an-issue-in-the-realtime-api.md:4]:
```suggestion
product_area: Platform services
```

**Review by sagscmi — CHANGES_REQUESTED** [review]
`MTM-56910` is actually a Realtime API ticket - it should be under Platform services / REST API.  I think the confusion was caused by having 2 components set on the ticket.

**Review by MWindel — APPROVED** [review]

File: content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Modified object default handling
+product_area: Device management & connectivity
+change_type:
+  - value: change-2c7RdTdXo4
+    label: Improvement
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md:7]: Feature

File: content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-1849
+version: 10.18.560.0
+---
+The default handling of object instance actions for LWM2M objects 3, 4 and 6 can now be turned off. In addition, the default server-side firmware update state machine for object 5 can be disabled.
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md:17]: Could you add this sentence?: 
Device's LWM2M Configuration UI can be used to enable/disable these settings.

File: content/change-logs/device-management/cumulocity-10-18-540-9-configurable-time-to-live-duration.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Configurable Time-To-Live duration
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-540-9-configurable-time-to-live-duration.md:3]:
```suggestion
title: LWM2M Time-to-Live duration for device operation realtime subscription
```

File: content/change-logs/device-management/cumulocity-10-18-540-9-configurable-time-to-live-duration.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3067
+version: 10.18.540.9
+---
+The LWM2M agent realtime subscription for device operations now has a configurable Time-To-Live (TTL) duration. The default TTL duration is 3 minutes.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-540-9-configurable-time-to-live-duration.md:17]:
```suggestion
The LWM2M agent realtime subscription for device operations now has a limited Time-To-Live (TTL) duration. The default TTL duration is 3 minutes. This setting may be configured on platform level.
```

File: content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Modified object default handling
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md:3]:
```suggestion
title: Option to disable LWM2M object default actions
```

File: content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md (hunk 3 above)
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md:7]:
```suggestion
    label: Feature
```

File: content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md (hunk 4 above)
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-560-0-modified-object-default-handling.md:17]:
```suggestion
The default handling of object instance actions for LWM2M objects 3, 4 and 6 can now be turned off. In addition, the default server-side firmware update state machine for object 5 can be disabled. These settings can be enabled/disabled in the device´s LWM2M  configuration in the UI. 
```

**Review by Oezge-Akin — APPROVED** [review]

**Pull request merged**